sections/header.liquid

```diff
@@ -618,6 +618,7 @@
       if (this.predictiveSearch && this.predictiveSearch.isOpen) return;
 
       if (scrollTop > this.currentScrollTop && scrollTop > this.headerBounds.bottom) {
+        if (this.preventHide) return;
```

Comment:
One of the other effect, I say "issue" in the video but it isn't one. Is that the sticky header will stay stick until you close the submenu: [video](https://screenshot.click/01-49-3tk40-3hzmt.mp4)

But since it's easy to close I don't see it being too much of an issue 🤔  Just sharing so we know all the behavioural changes with this.